### PR TITLE
Made workers dig tiles per face + display portals at game start + small fixes

### PR DESCRIPTION
**Pull request by hwoarangmy**

### CODE REVIEW

**Comment by akien-mga** [comment]
Nice work! I'll try to review the code tomorrow :)

File: source/creatureaction/CreatureActionDigTile.cpp [hunk 1]
```diff
@@ -99,19 +99,18 @@ bool CreatureActionDigTile::handleDigTile(Creature& creature, Tile& tileDig, Til
...
-                creature.receiveExp(digCoefGold * creature.getDigRate() / 20.0);
+                // Receive experience for digging gold
+                creature.receiveExp(creature.getDigRate() / 20.0);
                 break;
             }
             case TileType::gem:
             {
-                // TODO: wrong coef used here
-                static const double digCoefGem = ConfigManager::getSingleton().getDigCoefGold();
+                static const double digCoefGem = ConfigManager::getSingleton().getDigCoefGem();
```
Comment (akien-mga) [inline source/creatureaction/CreatureActionDigTile.cpp:108]: Nice catch :)

File: source/entities/Tile.h [hunk 2]
```diff
@@ -441,11 +441,11 @@ class Tile : public GameEntity
     bool canWorkerClaim(const Creature& worker);
     bool addWorkerClaiming(const Creature& worker);
     bool removeWorkerClaiming(const Creature& worker);
-    //! \brief Allows to lock the tile for the workers claiming it. Returns true if the worker could be added
-    //! and false otherwise
-    bool canWorkerDig(const Creature& worker);
-    bool addWorkerDigging(const Creature& worker);
-    bool removeWorkerDigging(const Creature& worker);
+    //! \brief Feels the tile vector with the available tiles the worker can
```
Comment (akien-mga) [inline source/entities/Tile.h:444]: s/Feels/Fills/ :)

**Comment by akien-mga** [comment]
Checked all commits and the code looks good (as well as the gameplay changes).

File: source/entities/Tile.h (hunk 2 above)
Comment (hwoarangmy) [inline source/entities/Tile.h:444]: lol. This one is huge ^^

**Pull request merged**